File: be/services/message.js
const mongoose = require('mongoose')

const Message = require("../models/message");

exports.createMessage = async (content, is_image, image_link, sender_id, receiver_id, created_at) =>{
    try {
        
        const newMessage = new Message({
            id: new mongoose.Types.ObjectId(),
            sender_id,
            receiver_id,
            created_at,
            content,
            is_image,
            image_link,
        })
        await newMessage.save()
        return newMessage
    } catch (error) {
        console.log('err create message')
    }
}

File: dashboard/src/main/java/com/mobile/dashboard/models/Message.java
package com.mobile.dashboard.models;

import lombok.*;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import java.util.Date;

@Data
@Document(collection = "messages")
@Getter
@Setter
@RequiredArgsConstructor
public class Message {
    @Id
    private String id;
    private String sender_id;
    private String receiver_id;
    private Date created_at;
    private String content;
    private Boolean is_image = false;
    private String image_link = "";

}
